File: imagepicker/src/main/java/com/gipl/imagepicker/models/ImageResult.java
package com.gipl.imagepicker.models;

import android.graphics.Bitmap;

/**
 * Created by Ankit on 01-Jun-19.
 */
public class ImageResult {
    private String sImagePath;
    private Bitmap bitmap;

    public ImageResult(String sImagePath, Bitmap bitmap) {
        this.sImagePath = sImagePath;
        this.bitmap = bitmap;
    }

    public String getsImagePath() {
        return sImagePath;
    }

    public Bitmap getImageBitmap() {
        return bitmap;
    }

    public void setImagePath(String sImagePath) {
        this.sImagePath = sImagePath;
    }

    public void setBitmap(Bitmap bitmap) {
        this.bitmap = bitmap;
    }
}

File: imagepicker/src/main/java/com/tap/imagepicker/models/ImageResult.kt
package com.tap.imagepicker.models

import android.graphics.Bitmap
import android.net.Uri

/**
 * Created by Ankit on 01-Jun-19.
 */
class ImageResult(private var sImagePath: String, var imageBitmap: Bitmap,var uri:Uri?=null) {
    fun getsImagePath(): String {
        return sImagePath
    }

    fun setImagePath(sImagePath: String) {
        this.sImagePath = sImagePath
    }

    fun setBitmap(bitmap: Bitmap) {
        imageBitmap = bitmap
    }
}